door.mjs

```diff
@@ -6,13 +6,40 @@ const doorEndpoint = process.env.UNIFI_DOOR_API;
 const doorAuthToken = process.env.UNIFI_DOOR_TOKEN;
 // Only use ACCESS events from these devices.
 const allowedDoorDevices = process.env.UNIFI_DOOR_DEVICES.split(' ');
+// Door access webhook API endpoint and API key
+const doorWebhookEndpoint = process.env.DOOR_ACCESS_WEBHOOK_ENDPOINT;
+const doorWebhookApiKey = process.env.DOOR_ACCESS_WEBHOOK_API_KEY;
 
 const doorHeaders = {
   Authorization: `Bearer ${doorAuthToken}`,
   Accept: 'application/json',
   'Content-Type': 'application/json',
 };
 
+const sendDoorEventsToWebhook = async (events) => {
+  if (!doorWebhookEndpoint || !doorWebhookApiKey) {
+    console.log('Door webhook endpoint or API key not configured, skipping webhook call');
```

Comment:
This will log on every event when it's disabled; probably too noisy for the intent.